docs/release-notes/8.0.33-25.upd.md

```diff
@@ -0,0 +1,19 @@
+# Percona XtraDB Cluster 8.0.33-25 Update (2023-08-)
+
+Percona XtraDB Cluster (PXC) supports critical business applications in your public, private, or hybrid cloud environment. Our free, open source, enterprise-grade solution includes the high availability and security features your business requires to meet your customer expectations and business goals.
+
+## Known issues
+
+If you use Galera Arbitrator (garbd), we recommend that you do not upgrade to 8.0.33 because `garbd-8.0.33` may cause synchronization issues. 
```

Comment:
Proposal:
`may cause synchronization issues and extensive usage of CPU resources.`

Additionally, maybe it is worthy to say something like this:
`If you already installed/upgraded to garbd-8.0.33 it is recommended to downgrade to garbd-8.0.32-24-2. To do this, uninstall percona-xtradb-cluster-garbd, then download manually percona-xtradb-cluster-garbd_8.0.32-24-2 from https://www.percona.com/downloads and install manually`

Reply:
fixed